certified-connectors/Revizto (Ireland)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1413 @@
+          "type": "string"
+        },
+        "id": {
+          "description": "The Revizto account ID",
+          "x-ms-summary": "The Revizto account ID",
+          "type": "integer"
+        },
+        "lastname": {
+          "description": "The user's last name",
+          "x-ms-summary": "The user's last name",
+          "type": "string"
+        },
+        "role": {
+          "description": "Account role for internal use",
+          "x-ms-summary": "Account role for internal use",
+          "type": "string",
+          "x-ms-visibility": "internal"
+        },
+        "uuid": {
+          "description": "The Revizto account UUID",
+          "x-ms-summary": "The Revizto account UUID",
+          "format": "uuid",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "uuid",
+        "firstname",
+        "lastname",
+        "role",
+        "email"
+      ]
+    },
+    "RoleItem": {
+      "description": "The project member's project role",
+      "x-ms-summary": "The project member's project role",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The project role ID",
+          "x-ms-summary": "The project role ID",
+          "example": 12345,
+          "type": "integer"
+        },
+        "name": {
+          "description": "The project role name",
+          "x-ms-summary": "The project role name",
+          "example": "Admin",
+          "type": "string"
+        },
+        "system": {
+          "description": "Predifined or custome role",
+          "x-ms-summary": "Predifined or custome role",
+          "example": 1,
+          "type": "integer",
+          "x-ms-visibility": "internal"
+        },
+        "uuid": {
+          "description": "The project role UUID",
+          "x-ms-summary": "The project role UUID",
+          "format": "uuid",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "uuid",
+        "name",
+        "system"
+      ]
+    },
+    "CommentsItem": {
+      "type" : "object"
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://ws.revizto.com/login?request=accessCode&",
+      "tokenUrl": "https://api.ireland.revizto.com/v5/oauth2",
+      "scopes": {
+        "DOCUMENT_READ DOCUMENT_WRITE": "DOCUMENT_READ DOCUMENT_WRITE"
+      }
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": [
+        "DOCUMENT_READ DOCUMENT_WRITE"
+      ]
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://revizto.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://revizto.com/en/privacy-policy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Collaboration;BIM"
```

Comment:
The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]

Reply:
Changed to "Collaboration,  Content and Files"